zookeeper-server/src/test/java/org/apache/zookeeper/server/quorum/QuorumPeerReadOnlyTest.java

```diff
@@ -0,0 +1,172 @@
+package org.apache.zookeeper.server.quorum;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+
+import org.apache.zookeeper.CreateMode;
+import org.apache.zookeeper.PortAssignment;
+import org.apache.zookeeper.ZooKeeper;
+import org.apache.zookeeper.KeeperException.RuntimeInconsistencyException;
+import org.apache.zookeeper.ZooDefs.Ids;
```

Comment:
useless import

Reply:
You can use `mvn checkstyle:check` to check whether you can pass the CI

Reply:
Thx
Is fixed now and should work in the new version